ovp.LucyBot.yml

```diff
@@ -557,3 +567,82 @@ workflowNavigation:
   - title: Contact Us
     icon: assets/img/icons/contact.png
     isContact: true
+
+iosNavigation:
+
+  - title: "Getting Started with the Player SDK for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/getting-started-ios.html
+  path: "/player/ios/getting"
+  discussionURL: null
+
+  - title: "Supported Devices for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/supported-devices-ios.html
+  path: "/player/ios/supported-devices-ios.html"
+  discussionURL: null
+
+- title: "Player Events for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/player-events-ios.html
+  path: "/player/ios/player-events-ios.html"
+  discussionURL: null
```

Comment:
Assuming you don't want to show the discussion pages on all these - other wise you can unset this field.

Reply:
unset meaning leave it out?

Reply:
yup!